packages/demo-cards/bio/embedded.js

```diff
@@ -3,11 +3,21 @@ import { precompileTemplate } from '@ember/template-compilation';
 import templateOnlyComponent from '@ember/component/template-only';
 import BoxelCardContainer from '@cardstack/boxel/components/boxel/card-container';
 import BoxelHeader from '@cardstack/boxel/components/boxel/header';
+import './embedded.css';
 
 export default setComponentTemplate(
-  precompileTemplate('<BoxelCardContainer><BoxelHeader @header="Bio" @editable={{true}}/></BoxelCardContainer>', {
-    strictMode: true,
-    scope: () => ({ BoxelCardContainer, BoxelHeader }),
-  }),
+  precompileTemplate(
+    `<BoxelCardContainer @displayBoundaries={{true}}>
+      <BoxelHeader>About Me</BoxelHeader>
```

Comment:
Not sure if the Boxel Card Container and Boxel Header for the contained cards should be on the template of each card, or if they should be on the CardSpaceLayout's isolated template.

Reply:
Hmmm. Have you tried doing on the isolated.js file of CardSpaceLayout? It seems like the visuals of the borders for these embedded views should really be controlled by the card that is inserting them.